ERROR: "__bad_udelay" [drivers/media/video/bt8xx/bttv.ko] undefined!
ERROR: "__bad_udelay" [drivers/media/dvb/bt8xx/dst.ko] undefined!
make[1]: *** [__modpost] Error 1

diff --git a/drivers/media/video/bt8xx/Kconfig b/drivers/media/video/bt8xx/Kconfig
index ce71e8e..15a3f6d 100644
--- a/drivers/media/video/bt8xx/Kconfig
+++ b/drivers/media/video/bt8xx/Kconfig
@@ -1,6 +1,6 @@
 config VIDEO_BT848
 	tristate "BT848 Video For Linux"
-	depends on VIDEO_DEV && PCI && I2C && VIDEO_V4L2 && INPUT
+	depends on VIDEO_DEV && PCI && I2C && VIDEO_V4L2 && INPUT && (!ARM || BROKEN)
 	select I2C_ALGOBIT
 	select VIDEO_BTCX
 	select VIDEOBUF_DMA_SG
